src/transformers/models/data2vec/convert_data2vec_text_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -31,11 +31,11 @@
     BertOutput,
     BertSelfAttention,
     BertSelfOutput,
+    Data2VecTextModel,
```

Comment:
I don't think `transformers.models.bert.modeling_bert` has `Data2VecTextModel`.

Reply:
Woopsie, wanted to put that in the line above. Thanks for the catch!